public/styles.css

```diff
@@ -29,6 +30,10 @@ blockquote {
   padding: 0;
   transform: translate(-50%, -50%);
   text-align: center;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
```

Comment:
We might not want this, but found that it selected text when clicking through the site.